filters/builtin/static.go

```diff
@@ -34,15 +36,19 @@ func (spec *static) CreateFilter(config []interface{}) (filters.Filter, error) {
 	}
 
 	webRoot, ok := config[0].(string)
+
 	if !ok {
 		return nil, fmt.Errorf("invalid parameter type, expected string for web root prefix")
 	}
-
 	root, ok := config[1].(string)
 	if !ok {
 		return nil, fmt.Errorf("invalid parameter type, expected string for path to root dir")
 	}
 
+	if !existsAndAccessible(root) {
```

Comment:
if ok, err := existsAndAccessible(root); !ok {
    log.Errorf("Invalid parameter for root path. File %s does not exist or is not accessible: %v", root, err)
}